tools/install.sh

```diff
@@ -48,6 +48,24 @@ case $os in
       echo
       echo "Please visit https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Installing-OpenBangla-Keyboard for Distrowise/Distro-specific Install Instructions."
       echo
-      ;;
+      
+      echo "Do you wish to visit the Distrowise Instructions website now?"
```

Comment:
How about: `Do you want to see the instructions for installing OpenBangla Keyboard in your system?`?

Reply:
Changed